arangod/Agency/Constituent.cpp

```diff
@@ -746,19 +744,41 @@ void Constituent::run() {
                 }
               }
             }
+            LOG_TOPIC("ddeae", DEBUG, Logger::AGENCY) << "Considering empty AppendEntriesRPC for follower "
+              << followerId << " needed: " << needed;
             if (needed) {
+              auto startTime = std::chrono::steady_clock::now();
               _agent->sendEmptyAppendEntriesRPC(followerId);
+              auto endTime = std::chrono::steady_clock::now();
+              if (endTime - startTime > std::chrono::milliseconds(100)) {
+                LOG_TOPIC("ddeeb", WARN, Logger::AGENCY)
+                  << "Call to sendEmptyAppendEntriesRPC took longer than 0.1 s: time needed: "
+                  << std::chrono::duration<double>(endTime - startTime).count();
+              }
```

Comment:
Isn't this somewhat expected that this can take a while?
At least `sendEmptyAppendEntriesRPC()` does network communication, which can take an arbitrary amount of time.
I am concerned this just adds more log spam in case things are already slow anyway. 
If we want to keep this log message, I would prefer setting the log level to INFO instead of WARN.